[2024-08-22T05:01:41.819+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Vehicle_Data_scrape_Pipeline_5.scrape_post_links manual__2024-08-22T05:01:40.804101+00:00 [queued]>
[2024-08-22T05:01:41.825+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Vehicle_Data_scrape_Pipeline_5.scrape_post_links manual__2024-08-22T05:01:40.804101+00:00 [queued]>
[2024-08-22T05:01:41.826+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 4
[2024-08-22T05:01:41.849+0000] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): scrape_post_links> on 2024-08-22 05:01:40.804101+00:00
[2024-08-22T05:01:41.853+0000] {standard_task_runner.py:60} INFO - Started process 501 to run task
[2024-08-22T05:01:41.861+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'Vehicle_Data_scrape_Pipeline_5', 'scrape_post_links', 'manual__2024-08-22T05:01:40.804101+00:00', '--job-id', '83', '--raw', '--subdir', 'DAGS_FOLDER/vehicle_data_scraping_pipeline.py', '--cfg-path', '/tmp/tmpxe91cpd2']
[2024-08-22T05:01:41.862+0000] {standard_task_runner.py:88} INFO - Job 83: Subtask scrape_post_links
[2024-08-22T05:01:41.900+0000] {task_command.py:423} INFO - Running <TaskInstance: Vehicle_Data_scrape_Pipeline_5.scrape_post_links manual__2024-08-22T05:01:40.804101+00:00 [running]> on host b761e1bbb455
[2024-08-22T05:01:42.156+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='sdeshan' AIRFLOW_CTX_DAG_ID='Vehicle_Data_scrape_Pipeline_5' AIRFLOW_CTX_TASK_ID='scrape_post_links' AIRFLOW_CTX_EXECUTION_DATE='2024-08-22T05:01:40.804101+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-08-22T05:01:40.804101+00:00'
[2024-08-22T05:03:41.990+0000] {taskinstance.py:2731} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 444, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 414, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 200, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 217, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/components/vehicle/data_scraping_components/post_links_extraction.py", line 68, in scrape_links
    upload_table('ikman_vehicle_post_links')
  File "/opt/airflow/dags/components/utils/database_table_creation.py", line 29, in upload_table
    cur.execute(f"""
psycopg2.errors.InsufficientPrivilege: could not open file "/var/lib/postgresql/csv/ikman_vehicle_post_links.csv" for writing: Permission denied
HINT:  COPY TO instructs the PostgreSQL server process to write a file. You may want a client-side facility such as psql's \copy.

[2024-08-22T05:03:42.021+0000] {taskinstance.py:1149} INFO - Marking task as UP_FOR_RETRY. dag_id=Vehicle_Data_scrape_Pipeline_5, task_id=scrape_post_links, execution_date=20240822T050140, start_date=20240822T050141, end_date=20240822T050342
[2024-08-22T05:03:42.036+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 83 for task scrape_post_links (could not open file "/var/lib/postgresql/csv/ikman_vehicle_post_links.csv" for writing: Permission denied
HINT:  COPY TO instructs the PostgreSQL server process to write a file. You may want a client-side facility such as psql's \copy.
; 501)
[2024-08-22T05:03:42.083+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-08-22T05:03:42.200+0000] {taskinstance.py:3312} INFO - 0 downstream tasks scheduled from follow-on schedule check
